src/components/media-player/ha-media-player-browse.ts

```diff
@@ -0,0 +1,699 @@
+                  ? html`
+                      ${mostRecentItem.children.map(
+                        (child) => html`
+                          <div
+                            class="child"
+                            .item=${child}
+                            @click=${this._navigate}
+                          >
+                            <div class="ha-card-parent">
+                              <ha-card
+                                style="background-image: url(${child.thumbnail})"
+                              >
+                                ${child.can_expand && !child.thumbnail
+                                  ? html`
+                                      <ha-svg-icon
+                                        class="folder"
+                                        .path=${mdiFolder}
+                                      ></ha-svg-icon>
+                                    `
+                                  : ""}
+                              </ha-card>
+                              ${child.can_play && !this._narrow
+                                ? html`
+                                    <div class="ha-card-copy">
+                                      <ha-svg-icon
+                                        class="play"
+                                        .item=${child}
+                                        .label=${this.hass.localize(
+                                          `ui.components.media-browser.${this.action}-media`
+                                        )}
+                                        .path=${this.action === "play"
+                                          ? mdiPlay
+                                          : mdiPlus}
+                                        @click=${this._actionClicked}
+                                      ></ha-svg-icon>
+                                    </div>
+                                  `
+                                : ""}
+                            </div>
+                            <div class="title">${child.title}</div>
+                            <div class="type">
+                              ${this.hass.localize(
+                                `ui.components.media-browser.content-type.${child.media_content_type}`
+                              )}
+                            </div>
+                          </div>
+                        `
+                      )}
+                    `
+                  : ""}
+              </div>
+            `
+          : html`
+              <ha-data-table
+                .columns=${this._columns}
+                .data=${mostRecentItem.children.map((child) => {
+                  return { icon: this.action, ...child };
+                }) as DataTableRowData[]}
+                .autoHeight=${true}
+                .showHeader=${false}
+                .id=${"media_content_id"}
+                @row-click=${this._rowClicked}
+              ></ha-data-table>
```

Comment:
Why do we use a `data-tabel` here? We don't use any of it's functions, can't we better use a `mwc-list`?

Reply:
```ts
              <mwc-list>
                ${mostRecentItem.children.map(
                  (child) => html`<mwc-list-item
                    @click=${this._actionClicked}
                    .item=${child}
                    graphic="icon"
                  >
                    <span>${child.title}</span>
                    <ha-svg-icon
                      slot="graphic"
                      .label=${this.hass.localize(
                        `ui.components.media-browser.${this.action}-media`
                      )}
                      .path=${this.action === "play" ? mdiPlay : mdiPlus}
                    ></ha-svg-icon
                  ></mwc-list-item>`
                )}
              </mwc-list>
```

Reply:
(and if you want the same styling, use `<li divider role="separator"></li>` and some css on the `mwc-list`)